feature-libs/product-configurator/rulebased/components/price/configurator-price.component.ts

```diff
@@ -41,7 +48,14 @@ export class ConfiguratorPriceComponent {
    * @return {string} - total price formula
    */
   get priceTotal(): string {
-    return '+ ' + this.formula.priceTotal?.formattedValue;
+    if (
+      this.formula.priceTotal !== undefined &&
+      this.formula.priceTotal.value >= 0
+    ) {
+      return '+' + this.formula.priceTotal?.formattedValue;
+    } else {
+      return this.formula.priceTotal?.formattedValue || '';
```

Comment:
Plz, explain what you expected to get here. 
`this.formula.priceTotal?.formattedValue || ''` syntax is not known.
According to debugger `this.formula.priceTotal?.formattedValue` will be returned.